components/trial_benefits_modal/trial_benefits_modal.tsx

```diff
@@ -181,20 +211,75 @@ const TrialBenefitsModal = (
         />
     )), [steps, handleOnClose]);
 
-    return (
-        <GenericModal
-            className='TrialBenefitsModal'
-            show={show}
-            id='trialBenefitsModal'
-            onExited={handleOnClose}
-        >
+    const trialJustStartedScreen = (declaration: Omit<TrialBenefitsModalStepProps, 'pageURL' | 'buttonLabel'>) => {
+        const {id, title, description, svgWrapperClassName, svgElement, bottomLeftMessage} = declaration;
```

Comment:
What about:
```
const trialJustStartedScreen = ({id, title, description, svgWrapperClassName, svgElement, bottomLeftMessage}:  Omit<TrialBenefitsModalStepProps, 'pageURL' | 'buttonLabel'>) => (
```

Reply:
In a recent change (https://github.com/mattermost/mattermost-webapp/pull/9803) `buttonLabel` prop was added to `TrialBenefitsModalStep`, which is actually only required when `pageURL` is provided.
So maybe should set `buttonLabel` also as optional, then wont need the Omit here.